js/language/scss.js

```diff
@@ -0,0 +1,836 @@
+/**
+ * SCSS patterns
+ * 
+ * @author Fabio Bucci Di Monte
+ * @version 1.0.1
+ */
+!function(){
+
+    'use strict';
+
+    if(!Rainbow){
```

Comment:
Why would Rainbow not be included here? Also checking for it directly will actually trigger an error if it doesn't exist